PowerStrip.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include "PowerStrip.h"
+
+PowerStrip::PowerStrip() : 
+numOutlets(3),
+maxSurgeProtection(2400.f),
+numPopulatedOutlets(0)
+{
+    std::cout << "PowerStrip" << std::endl; 
+    outlet1.outletNum = 1;    
+    outlet2.outletNum = 2;    
+    outlet3.outletNum = 3;
+}
+
+PowerStrip::~PowerStrip()
+{
+    std::cout << "PowerStrip object destroyed." << std::endl;
+}
+
+int PowerStrip::insertPlug(int outletNum) 
+{
+    if (outlet1.outletNum == outletNum) outlet1.plugInstalled = true;
+    else if (outlet2.outletNum == outletNum) outlet2.plugInstalled = true;
+    else if (outlet3.outletNum == outletNum) outlet3.plugInstalled = true;
+    else 
+    {
+        std::cout << "There is no outlet number " << outletNum << " in power strip";
+        return 0;
+    }
```

Comment:
If you're going to use curly braces in an `if/else`, use them for every `if/else if/`
- [ ] revise